product-specifications/example-2.md

```diff
@@ -0,0 +1,21 @@
+# Interactive Animal Identification Tool (IAIT)
+
+This tool will help users identify animals by asking a series of questions and then provide a list of most likely animals based on the user's responses. This tool will be used by wildlife conservationists to identify animals they encounter.
+
+## User Flow
+
+1. Users answer a series of questions about the observed animal:
+    - What color is the animal?
```

Comment:
I feel like colour may create a paradox, since there are people who do not have a wide colour range/knowledge of colours or are colourblind, but at the same time, colours may be vital in telling close species of animals apart, especially in later updates when more specific animal species will be added.